doc/user/PLUGINS.md

```diff
@@ -0,0 +1,43 @@
+## Plugins
+
+F3D officially supports plugins for certain file formats. If you installed F3D using the binary release, there's no need to manually load these plugins when opening your file. F3D will load them automatically.
+
```

Comment:
remove empty line

Reply:
the paragraph break in line 4? any reason for this? i feel it's more readable with the empty line.